src/query.rs

```diff
@@ -18,4 +19,37 @@ impl Query {
     fn __repr__(&self) -> PyResult<String> {
         Ok(format!("Query({:?})", self.get()))
     }
+
+    /// Construct a Tantivy's TermQuery
+    #[staticmethod]
+    #[pyo3(signature = (schema, field_name, field_value, index_option = "position"))]
+    pub(crate) fn new_term_query(
```

Comment:
I would suggest just `term_query`.

Reply:
Thanks for the suggestion. `term_query` sounds more concise.

Reply:
Renamed in https://github.com/quickwit-oss/tantivy-py/pull/175/commits/e02b47986fc9c6f1de836ec49a9922a83e6ffd85.